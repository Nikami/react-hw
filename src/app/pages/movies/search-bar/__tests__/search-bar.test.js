import React from 'react';
import { shallow } from 'enzyme';
import SearchBar from '../index';

describe('SearchBar', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<SearchBar />);
    expect(wrapper).toMatchSnapshot();
  });
});
