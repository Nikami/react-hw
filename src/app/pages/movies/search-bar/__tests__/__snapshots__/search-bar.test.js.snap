// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar matches snapshot 1`] = `
<WithStyles(BgContainer)>
  <_default />
  <form
    autoComplete="off"
  >
    <WithStyles(Typography)
      className="text-uppercase"
      component="h6"
      gutterBottom={true}
      variant="h6"
    >
      Find your movie
    </WithStyles(Typography)>
    <TextField
      className="search-field"
      id="search"
      margin="none"
      placeholder="Search"
      required={false}
      select={false}
      variant="standard"
    />
    <div
      className="search-actions"
    >
      <div
        className="search-actions__toolbar"
      >
        <WithStyles(Typography)
          className="text-uppercase"
          component="span"
          variant="subtitle2"
        >
          SearchÂ By
        </WithStyles(Typography)>
        <WithStyles(Button)
          color="primary"
          size="small"
          variant="contained"
        >
          Title
        </WithStyles(Button)>
        <WithStyles(Button)
          color="secondary"
          size="small"
          variant="contained"
        >
          Genre
        </WithStyles(Button)>
      </div>
      <WithStyles(Button)
        color="primary"
        size="medium"
        variant="contained"
      >
        Search
      </WithStyles(Button)>
    </div>
  </form>
</WithStyles(BgContainer)>
`;
